package.json

```diff
@@ -31,6 +31,20 @@
   },
   "preferences": [
     {
+      "name": "PROTOCOL",
+      "title": "Ideatown protocall",
+      "description": "protocall",
+      "type": "string",
+      "value": "https://"
+    },
+    {
+      "name": "HOSTNAME",
+      "title": "Ideatown hostname",
+      "description": "hostname",
+      "type": "string",
+      "value": "ideatown.dev"
```

Comment:
This should probably be `ideatown.dev:8000` until we replace it with a production URL. 

Also might be better to just have a single `BASE_URL` pref like @6a68 mentioned. Looks like `HOSTNAME` is necessary for `getCookiesFromHost`, but `PROTOCOL` & `HOSTNAME` get concatenated everywhere else. Might be worth simplifying most places to extract `HOSTNAME` in the place it's needed.